R/msaAttribute.R

```diff
@@ -91,7 +91,8 @@ msaAttribute <- function(jaspResults, dataset, options, ...) {
       jaspResults[["AAAtableGraphs"]] <- createJaspContainer(gettext("Attributes Agreement Analysis"))
       jaspResults[["AAAtableGraphs"]]$position <- 16
     }
-    jaspResults[["AAAtableGraphs"]] <- .aaaTableGraphs(ready = ready, dataset = dataset, measurements = measurements, parts = parts, operators = operators, options =  options, standards = standards)
+    jaspResults[["AAAtableGraphs"]] <- .aaaTableGraphs(ready = ready, dataset = dataset, measurements = measurements,
```

Comment:
This doesn't look quite right, but I guess this was already the case?
As in, two lines above you do
```r
jaspResults[["AAAtableGraphs"]] <- createJaspContainer(gettext("Attributes Agreement Analysis"))
```
and here you do
```r
jaspResults[["AAAtableGraphs"]] <- .aaaTableGraphs(...)
```
which means that you overwrite everything that was in `jaspResults[["AAAtableGraphs"]]` before. I mean, it probably works because it already worked before, but it's not very pretty.

Reply:
Yes, I think there is quite a bit in the measurement systems analysis that is not pretty, I would keep this for now, as I have a bigger rewrite of this part planned for early next year...